draccus/argparsing.py

```diff
@@ -78,9 +82,16 @@ def _set_dataclass(
             default = dataclass if default is None else default
             dataclass = type(dataclass)
 
-        new_wrapper = dataclass_wrapper_class(dataclass, default=default)
+        new_wrapper = dataclass_wrapper_class(dataclass, default=default, preferred_help=self.preferred_help)
         new_wrapper.register_actions(parser=self.parser)
 
+    def _assert_preferred_help(self):
+        """Checks that `self.prefer_help` is valid."""
+        if self.preferred_help not in {"inline", "above", "below"}:
```

Comment:
does it make sense to break out a stringenum?

Reply:
Feels like more overhead? Then if you're overloading `draccus.wraps()` it's an extra import (`from draccus import Help; draccus.wrap(Help.INLINE)`? Your call though

Reply:
i was thinking stringenum so you could choose between the two (HF does things this way frequently)

Reply:
Fixed... I think?

Reply:
i meant something much simpler really. I went in and did it. https://github.com/dlwh/draccus/pull/9/commits/1f6ca95387b6505f3d1e5697125a4c0e2c6159af#diff-a524e53291b158cc810ec980f25dfcf430909d75a6e1fc0de59b2cfe4a1656b8

Reply:
Oh I totally missed this - thanks David!